Sources/PointFree/Account/Account.swift

```diff
@@ -630,11 +630,14 @@ private let subscriptionPaymentInfoView = View<Stripe.Subscription> { subscripti
   ]
 }
 
+public func format(cents: Stripe.Cents) -> String {
+  let dollars = NSNumber(value: Double(cents.rawValue) / 100)
+  return currencyFormatter.string(from: dollars)
+    ?? NumberFormatter.localizedString(from: dollars, number: .currency)
+}
+
 private func totalAmount(for subscription: Stripe.Subscription) -> String {
-  let totalCents = subscription.plan.amount.rawValue * subscription.quantity
-  let totalDollars = NSNumber(value: Double(totalCents) / 100)
-  return currencyFormatter.string(from: totalDollars)
-    ?? NumberFormatter.localizedString(from: totalDollars, number: .currency)
+  return format(cents: .init(unwrap: subscription.plan.amount.rawValue * subscription.quantity))
```

Comment:
would `Tagged: Numeric` make this simpler?

Reply:
Yup. I can maybe get Web and Point-Free usin Tagged after mergin this.